src/main/scala/mesosphere/mesos/Constraints.scala

```diff
@@ -25,6 +25,21 @@ object Constraints {
     case _      => default
   }
 
+  private def getValueString(attribute: Attribute): String = attribute.getType match {
+    case Value.Type.SCALAR =>
+      java.text.NumberFormat.getInstance.format(attribute.getScalar.getValue)
+    case Value.Type.TEXT =>
+      attribute.getText.getValue
+    case Value.Type.RANGES =>
+      val s = attribute.getRanges.getRangeList.asScala
+        .map(r => s"${r.getBegin.toString}-${r.getEnd.toString}")
+        .mkString(",")
+      s"[$s]"
+    case Value.Type.SET =>
+      val s = attribute.getSet.getItemList.asScala.mkString(",")
```

Comment:
I think we should sort the set. `"a,b" != "b,a"`
With sorting the use at least know how to specify this.
This should also be part of the documentation.